sources/packages/backend/libs/integrations/src/object-storage/models/object-storage.models.ts

```diff
@@ -0,0 +1,11 @@
+export interface StorageObject {
+  key: string;
+  body: Buffer;
+  contentType: string;
+}
+
+export interface GetObjectResult {
+  contentLength: number;
+  contentType: string;
+  body: NodeJS.ReadableStream;
+}
```

Comment:
What about following DTO convention here?

Reply:
I do not see a reason to be honest. Even if it is not wrong to call it a DTO we usually do not use it from service to service models, at least that is my perception.
Even for services like BCeID I believe that we use models instead DTO conventions.